test/TestApps/PushNotificationsTestApp/main.cpp

```diff
@@ -252,7 +252,7 @@ bool VerifyForegroundHandlerFails()
     try
     {
         // Register is already called in main
-        PushNotificationManager::Default().PushReceived([](const auto&, PushNotificationReceivedEventArgs const& args) {});
+        PushNotificationManager::Default().PushReceived([](const auto&, PushNotificationReceivedEventArgs const& /*args*/) {});
```

Comment:
>* [](http://example.com/codeflow?start=114&length=1)

Mixing patterns on the same line.  The first parameter is nameless instead of commented out.

Reply:
The latter was a compiler warning at /W4 (was /W3) so I fixed it. I didn't notice the missing parameter. Is that supposed to be "sender"?

Reply:
Most likely.  To quiet this particular warning you can either comment out the name of the variable (so there is nothing to be unused) or you can delete the name entirely (same effect).  Both are valid options.  I prefer the commented out name in most cases.  My nitpick was just that this line is mixing both patterns instead of sticking to one.

Reply:
I prefer commenting out too. 'Cause today's unused parameter is tomorrow's PITA need-to-use-what-was-the-bloody-thing-called...

My inner OCD rose to the occasion so it's not commented out for consistency :-)